docs/resources/security_monitoring_suppression.md

```diff
@@ -10,7 +10,18 @@ description: |-
 
 Provides a Datadog Security Monitoring Suppression API resource. It can be used to create and manage Datadog security monitoring suppression rules.
 
-
+## Example Usage
+
+```terraform
+resource "datadog_security_monitoring_suppression" "my_suppression" {
+  name              = "My suppression"
+  description       = "Suppression for low severity cloudtrail signals from test environments limited to 2024"
```

Comment:
```suggestion
  description       = "Suppression for low severity CloudTrail signals from test environments limited to 2024"
```

Reply:
Done in [e4e6107](https://github.com/DataDog/terraform-provider-datadog/pull/2288/commits/e4e6107db763b99e2807feb8bd3ce1398ffd9a9c)